ui/modules/apps/BeamMP-Session/app.css

```
@@ -12,7 +12,14 @@
 		justify-content: center;
 		flex-flow: row nowrap;
 	}
-	.block {
+    .info-container {
+        display: flex;
+		flex-flow: row nowrap;
+        height: 100%;
+        flex: 1 1 auto;
```

This is probably causing that weird spacing issue when there are no server name.

Maybe this would work:

```suggestion
        flex: 0 1 auto;
```

Or this:

```suggestion
        flex: 0 0 auto;
```